components/ProductDetails.js

```diff
@@ -4,99 +4,79 @@ import { faStar } from '@fortawesome/fontawesome-free-solid';
 import { Colors } from '../common/theme';
 
 const Wrapper = styled.div`
-  margin: 10em auto auto auto;
-  display: flex;
-  justify-content: center;
-  padding: 1.25rem 2.5rem;
-  text-transform: capitalize;
-`;
-const ProductContainer = styled.div`
   display: flex;
+  flex-direction: row;
   align-items: center;
   justify-content: center;
+  text-transform: capitalize;
   .price {
     font-weight: bold;
   }
+  img {
+    max-height: 30rem;
+  }
   @media only screen and (max-width: 580px) {
-    display: flex;
     flex-direction: column;
+    img {
+      max-height: 20rem;
+    }
   }
 `;
-const ProductImg = styled.div`
-  text-align: center;
-  max-width: 30em;
-`;
 const ProductInfo = styled.div`
   max-width: 50em;
   @media only screen and (max-width: 1100px) {
```

Comment:
Maybe we also can unify media breakpoints if it makes a sense. Since I have seen one commonly used is 580px but there are several other in place. I do not know if it was made by purpose or not.